lib/plug_rails_cookie_session_store/message_encryptor.ex

```diff
@@ -52,6 +52,30 @@ defmodule PlugRailsCookieSessionStore.MessageEncryptor do
     end
   end
 
+  @doc """
+  Encrypts and signs a message.
+  """
+  def encrypt_and_authenticate(message, secret, cipher \\ :aes_gcm)
```

Comment:
```suggestion
  def encrypt_and_authenticate(message, secret, cipher \\ :aes_256_gcm)
```

Rails seems to use AES 256 GCM: https://github.com/rails/rails/pull/28132/files#diff-744c15344fa1f284281b429673de936cR231


which seems to be a cypher type in :crypto:

https://github.com/erlang/otp/blob/master/lib/crypto/src/crypto.erl#L495

Reply:
@cconstantin can you take a look at this? :)

Reply:
@gstokkink @4xposed what this is missing is a passing test for rails 5.2. I got stuck on that and never had time to get back and complete. Any chance you can contribute that?